static/js/popovers.js

```diff
@@ -510,6 +510,58 @@ exports.register_click_handlers = function () {
         e.stopPropagation();
         e.preventDefault();
     });
+    $('body').on('click', '.view_edit_history', function (e) {
+        var msgid = $(e.currentTarget).data('msgid');
+        var row = current_msg_list.get_row(msgid);
+        var message = current_msg_list.get(rows.id(row));
+
+        popovers.hide_actions_popover();
+        $('#message-edit-history').modal("show");
+        channel.get({
+            url: "/json/message_edit_history",
+            data: {"message_id": JSON.stringify(message.id)},
+            success: function (data) {
+                var message_history = data.message_history;
+                _.each(message_history, function (msg) {
+                    if (_.indexOf(message_history, msg) === message_history.length-1) {
+                        msg.timestamp = message.timestamp;
+                        msg.prev_rendered_content = msg.prev_rendered_content.slice(3, -4);
```

Comment:
Yeah. Because there is some discountinuity in `prev_rendered_content` for the original message.
`Hello<span class="highlight_text_inserted">, all</span>`
`<p>Hello</p>`.
These `<p>` tags were causing irregular space in message history modal. See screenshot 
![screen shot 2016-07-27 at 1 34 17 am](https://cloud.githubusercontent.com/assets/6594924/17153451/7807b072-539a-11e6-98b6-e454a13b9abc.png)